codemon/CodemonReg.py

```diff
@@ -6,22 +6,31 @@
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 
 def codemonReg():
-    print(ROOT_DIR)
-    path = os.path.join(ROOT_DIR, "codemon")
+    #print(ROOT_DIR)
+    prefix = '.' if os.name != 'nt' else ''
+    file_name = prefix + "codemon"
+    path = os.path.join(ROOT_DIR, file_name)
     if not os.path.exists(path):
         os.makedirs(path)
-        FILE_ATTRIBUTE_HIDDEN = 0x02
-        ret = ctypes.windll.kernel32.SetFileAttributesW(path, FILE_ATTRIBUTE_HIDDEN)
-    meta = ""
-    name = input("Name : ")
-    cf_user = input("CF Username : ")
-    cf_pass = input("CF Password : ")
-    git_user = input("GitHub Username : ")
-    git_pass = input("GitHub Password : ")
-    meta += name + "\n" + cf_user + "\n" + cf_pass + "\n" + git_user + "\n" + git_pass
-    full_filename = os.path.join(ROOT_DIR, "codemon", "meta.txt")
+        if os.name == 'nt':
+            FILE_ATTRIBUTE_HIDDEN = 0x02
+            ret = ctypes.windll.kernel32.SetFileAttributesW(path, FILE_ATTRIBUTE_HIDDEN)
+
+    print("Codemon User Registration")
+    print("\nPlease enter the following data. Your sensitive information is collected for making your life easier while using Codemon and will be stored on your local machine. It won't be stored or used by Codemon developers for any purpose. ")
+    name = input("\nEnter Name : ")
+    codeforces_username = input("Codeforces username (without @) : ")
+    codeforces_password = input("Codeforces password: ")
+    github_username = input("Github username: ")
```

Comment:
should be - `Github username (without @) `
Also, currently user can view the typed password, can you make it such that the password is viewed as ` ******` on CLI ?